core/arch/arm/include/kernel/vfp.h

```diff
@@ -91,7 +90,7 @@ void vfp_lazy_save_state_init(struct vfp_state *state);
  * If VFP was enabled when vfp_lazy_save_state_init() was called: save rest
```

Comment:
"... called, or @force_save is true: save..." perhaps? (although it's quite obvious)